dali/pipeline/operators/python_function/dltensor_function.cc

```diff
@@ -19,13 +19,25 @@ namespace dali {
 
 DALI_SCHEMA(DLTensorPythonFunctionImpl)
     .AddParent("PythonFunctionImplBase")
+    .AddOptionalArg("synchronize_stream", "", true)
```

Comment:
missing documentation for the option. Is it on purpose? You could change the parent:
```
DALI_SCHEMA(DLTensorPythonFunctionImpl)
   .AddParent("DLTensorPythonFunction")
   .AddParent("PythonFunctionImplBase")
   ...
   .MakeInternal();
```
to avoid redefining the `synchronize_stream` here.

Reply:
The `*Impl` operators are internal and the documentations isn't crucial there. `*Impl` operators need to have different schemes than the exposed python operators because they have different type of arguments (`function` vs `function_id`).